### PR TITLE
[DOC-17] Gov 2.0 updates 

### PR DESCRIPTION
**Pull request by philburrrt**
'Governance and Operations' details Gov 2.0 structure.
YIP related content has been generalized to 'proposal' to include info on YDPs

Also updated the team list with publicly available information

### CODE REVIEW

File: resources/team.md [hunk 1]
```diff
@@ -4,51 +4,56 @@
...
+
+
+## Multisig
+
+|Team Member| Twitter| Address|
+|-----------|--------|-------|
+|Alexey Karpenko|[@kaplansky1](https://twitter.com/kaplansky1/status/1285427247286046725)|0x74630370197b4c4795bFEeF6645ee14F8cf8997D|
+|Daryl Lau|[@daryllautk](https://twitter.com/Daryllautk/status/1285434908383444992)|0x99BC02c239025E431D5741cC1DbA8CE77fc51CE3|
+|devops199fan|[@devops199fan](https://twitter.com/devops199fan/status/1285430347954622464)|0x48f2bd7513da5Bb9F7BfD54Ea37c41650Fd5f3a3|
+|Ryan Watckins|[@RyanWatkins_](https://twitter.com/RyanWatkins_)|0x757280Bd46fC5B1C8b85628E800c443525Afc09b|
```
Comment (listonjesse) [inline resources/team.md:49]: I think there is a typo in his name here. Ryan Watkins

Comment (philburrrt) [reply resources/team.md:49]: Good catch, will fix after more comments come in.

**Comment by philburrrt** [comment]
Merged my branch with new PR

**Comment by lehnberg** [comment]
@philburrrt looks like you accidentally added forestry files to this with that last commit? can you delete the `.forestry/settings.yml` file before I merge?

**Comment by philburrrt** [comment]
@lehnberg Woops. Fixed that and sync'd resolved conflicts with david's image commit.

**Pull request closed without merging**